bindata/network/multus-admission-controller/001-service.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: multus-admission-controller
+  namespace: openshift-multus
+  labels:
+    app: multus-admission-controller
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: multus-admission-controller-secret
+spec:
+  ports:
+  - port: 443
+    targetPort: 443
```

Comment:
I'd recommend adding this as port 6443, and running the admission controller as non-root.

Reply:
Changed.